src/sagas/projects.js

```diff
@@ -16,13 +16,14 @@ import {
   gistNotFound,
   projectCreated,
   projectLoaded,
+  changeCurrentProject as __changeCurrentProject,
```

Comment:
I don’t think we’ll need this import if we just create a new `projectRestoredFromLastSession` action, but FWIW I think it’s better to disambiguate more explicitly, e.g. `changeCurrentProject as createChangeCurrentProjectAction`